setup.py

```diff
@@ -11,7 +11,10 @@
     'ftw.testing',
     'plone.app.testing',
     'plone.testing',
-    'pandas==0.22.0',
+    # later versions of pandas require python-dateutil>=2.5.0
+    'pandas < 0.23.0',
```

Comment:
General comment to version constraints:
In the setuptools version comparison function, alpha versions are sorted before betas, betas before stable releases. Thus:
`0.23.0a < 0.23.0`

So if a `pandas==0.23.0a1` would be released, we'd install it.

That's why I usually use:
```suggestion
    'pandas < 0.23a',
```

Reply:
I'll put it to my notes. I'm going to merge it like that anyway because it is very unlikely that they will do that and I activated nightly testing (so I'd notice immediately).